tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -482,6 +483,18 @@ def __init__(self,
       tf_logging.info("Running against TensorRT version %s" % ".".join(
           [str(x) for x in loaded_version]))
 
+    # Load TensorRT plugins.
+    try:
+      plugin_lib = ctypes.CDLL("libnvinfer_plugin.so")
```

Comment:
Why do we need to load it here? Can we just link the library into convert_nodes?